framework/src/auxkernels/VectorVariableMagnitudeAux.C

```diff
@@ -0,0 +1,32 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "VectorVariableMagnitudeAux.h"
+
+registerMooseObject("MooseApp", VectorVariableMagnitudeAux);
+
+InputParameters VectorVariableMagnitudeAux::validParams() {
+  InputParameters params = AuxKernel::validParams();
+  params.addClassDescription("Creates a field consisting of the magnitude of a "
+                             "coupled vector variable.");
+  params.addRequiredCoupledVar(
+      "vector_variable", "The variable from which to compute the component");
+  return params;
+}
```

Comment:
Different formatting. I'm going to get buzzed by `clang-format` but that's fine :smile:

Reply:
I'll fix the formatting when this is merged

Reply:
sounds good; i won't touch this then